README.md

```diff
@@ -179,6 +179,9 @@ To run tests, run the following command:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Deployment
+ - github pages
+
 <!-- AUTHORS -->
 
 ## 👥 BANCUNGUYE66 <a name="authors"></a>
```

Comment:
Please fix the MIT link as it is currently broken and leads to a 404 page instead of the license for the project. in order to improve your documentation.

Reply:
- [ ] Again here as requested by the previous reviewer, kindly fix the broken link to the MIT license agreement. You could download it from [here](https://github.com/microverseinc/readme-template)